README.md

```diff
@@ -1,2 +1,11 @@
 # runveteransmarathon
 Website for runveteransmarathon.com
+
+
+Website content, currently being maintained in Wordpress and Google Docs:
+
+https://drive.google.com/drive/folders/1snDeKYPr5dy88d0ZFmtm_NmglqFth-Qv?usp=sharing
```

Comment:
This repo is public. Do we want this in here?

Reply:
People could request access... but they won't be able to see it
It doesn't need to be in here tho and can easily just be a bookmark for Sam and us.

![image](https://user-images.githubusercontent.com/24508237/149666207-9510a353-58a0-4fed-82e8-749165a06310.png)